packages/ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@harnessio/ui",
   "description": "Harness Canary UI component library",
-  "version": "0.0.127",
+  "version": "0.0.129",
```

Comment:
bumping version in a separate PR is recommended for two reasons:
1. we can have multiple changes in one version bump
2. if order of merging PRs is different, version bumps conflict very easily